EIPS/eip-2070.md

```diff
@@ -22,38 +22,40 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Berl
 
 - TBD
 
-### Accepted EIPs
+### EFI accepted EIPs
 
-- TBD
+- [EIP 2537](https://github.com/ethereum/EIPs/pull/2537): BLS12-381 curve operations
+- [EIP 2315](https://github.com/ethereum/EIPs/pull/2576): Simple Subroutines for the EVM 
 
-### Tentatively Accepted EIPs
-
-- [EIP-663](https://eips.ethereum.org/EIPS/eip-663): Unlimited SWAP and DUP instructions
-- [EIP-1057](https://eips.ethereum.org/EIPS/eip-1057): ProgPoW, a Programmatic
-  Proof-of-Work
-  - There is a
-    [pending audit](https://medium.com/ethereum-cat-herders/progpow-audit-goals-expectations-75bb902a1f01),
-    above and beyond standard security considerations, that should be evaluated
-    prior to inclusion.
-- [EIP-1380](https://eips.ethereum.org/EIPS/eip-1380): Reduced gas cost for call to self
-- [EIP-1702](https://eips.ethereum.org/EIPS/eip-1702): Generalized account versioning scheme
-- [EIP-1962](https://eips.ethereum.org/EIPS/eip-1962): EC arithmetic and pairings with runtime definitions
-  - replaces EIP-1829
-- [EIP-1985](https://eips.ethereum.org/EIPS/eip-1985): Sane limits for certain EVM parameters
-- [EIP-2045](https://eips.ethereum.org/EIPS/eip-2045): Particle gas costs for EVM opcodes
-- [EIP-2046](https://eips.ethereum.org/EIPS/eip-2046): Reduced gas cost for static calls made to precompiles
 
-### Proposed EIPs
+
+### Tentatively accepted for upgrade
```

Comment:
IMO we should remove this section altogether, because we have the EFI list that keeps track of those.

Reply:
Sure.
So, we will have two clearly defined section  -  Accepted EIPs & Proposed EIPs